script.js

```diff
@@ -12,7 +12,7 @@ $(window).on('load', function () {
                             <td>` + element.date + `</td>
                             <td>` + (element.price === "" ? '-' : element.price) + `</td>
                             <td> <span style="color:`+ (element.ball < 0 ? '#FF4500': 'black' ) +`">` + (element.note === "" ? '-' : element.note) + `</span></td>
-                            <td>` + (element.thanks === "" ? '-' : element.thanks) + `</td>
+                            <td><button type="button" class="btn btn-sm btn-danger" data-toggle="popover" title="" data-content="`+ (element.thanks === "" ? '-' : element.thanks)  +`">Thanks</button></td>
```

Comment:
This "Thanks" popover only applies when we have new balls in. It is not needed when some balls wear out. One alternative I come up with is to add an if conditional statement judged by if the element.ball < 0 or not.

Reply:
& I prefer 'btn-outline-info' instead of 'btn-danger' :)

Reply:
Updated